core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java

```diff
@@ -57,9 +55,9 @@ String lineEnd() {
         private Writer writer;
         private boolean header = true;
         private char separator = ',';
-        private String lineEnd = CSVWriter.DEFAULT_LINE_END;
-        private char escapeChar = CSVWriter.DEFAULT_ESCAPE_CHARACTER;
-        private char quoteChar = CSVWriter.NO_QUOTE_CHARACTER;
+        private String lineEnd = "\n";
+        private char escapeChar = '"';
+        private char quoteChar = '\u0000'; // TODO review
```

Comment:
In OpenCSV, this option is used to put quotes around every cell, which is (A) according to the standard, (B) not widely followed, and (C) makes the file size much bigger. I don't think we should be quoting by default.  
I'm not an expert, but a quick google suggests that the unicode null character (in C++ land at least) used 2 bytes, so I would prefer to write nothing around the cells if that is possible.